### PR TITLE
3Create melvin.is-an.app

### PR DESCRIPTION
**Pull request by MartinTintin3**
### Link to Website

Link: 

melvin4life.com
melvin.is-an.app

---

- [ ] Toggle this checkbox to re-run the stargazer checking

### CODE REVIEW

File: domains/melvin.is-an.app [hunk 1]
```diff
@@ -0,0 +1,13 @@
+  "$schema": "../schemas/domain.schema.json",
```
Comment (tarampampam) [inline domains/melvin.is-an.app:1]: The wrong file extension and missing `{` at the file start

File: domains/melvin.is-an.app [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "description": "Some unblocked games",
+  "domain": "is-an.app",
+  "subdomain": "melvin",
+  "owner": {
+    "repo": "<https://URL/to/the/repository/with/subdomain/content/sources>",
+    "email": "<no>"
```
Comment (tarampampam) [inline domains/melvin.is-an.app:7]: Missing owner info

File: domains/melvin.is-an.app [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  },
+  "record": {
+    "CNAME": "melvin4life.com",
```
Comment (tarampampam) [inline domains/melvin.is-an.app:10]: Why do you want a subdomain if you already have a domain?

**Review by tarampampam — CHANGES_REQUESTED** [review]

**Comment by MartinTintin3** [comment]
Because I need different aliases for the original domain.

**Comment by tarampampam** [comment]
But why?

File: domains/melvin.is-an.app (hunk 3 above)
Comment (MartinTintin3) [reply domains/melvin.is-an.app]: Because I need different aliases for the original domain.

**Comment by MartinTintin3** [comment]
Because it's a website with games, one of the domains can get blocked. Then people can use another domain...

**Comment by tarampampam** [comment]
It sounds a bit strange but ok, approved

**Review by tarampampam — APPROVED** [review]

**Comment by tarampampam** [comment]
@MartinTintin3  Looks like an SSL problem on your origin server. May we can try to enable CF proxy?

**Comment by MartinTintin3** [comment]
Ok

**Comment by MartinTintin3** [comment]
Oh waaait I haven't added an ssl certificate for this url

**Comment by tarampampam** [comment]
One second...

**Comment by tarampampam** [comment]
Oh ok

**Comment by MartinTintin3** [comment]
Alright works now. Thx for reminding me

**Comment by tarampampam** [comment]
Not a problem dude

**Pull request merged**